tests/components/homekit/test_homekit.py

```diff
@@ -165,13 +163,11 @@ def test_generate_aid():
 
 async def test_homekit_setup_advertise_ip(hass, hk_driver):
     """Test setup with given IP address to advertise."""
-    homekit = HomeKit(
-        hass, BRIDGE_NAME, DEFAULT_PORT, "0.0.0.0", {}, {}, None, "192.168.1.100"
-    )
+    homekit = HomeKit(hass, BRIDGE_NAME, DEFAULT_PORT, "0.0.0.0", {}, {}, None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)